033-OSSDatabaseMigration/Coach/README.md

```diff
@@ -0,0 +1,23 @@
+# What The Hack - Intro To OSS DB Migration to Azure OSS DB
+## Introduction
+The tips in this coach's guide are intended to help you guide the attendees as they go through the challenges. You should not immediately give them out. It is okay for the attendees to struggle for a bit before arriving at the solution. This is often the best way for them to learn. Use your best judgement for deciding when to give them additional help.
```

Comment:
Add a link to the Lectures presentation here.

Also add some text for any pre-event set up that's needed and even some instructions on zipping up the Student/Resources files and putting them into the Teams files tab.

Reply:
Fixed